include/fluent-bit/flb_info.h.in

```diff
@@ -29,4 +29,20 @@
 @FLB_BUILD_FLAGS@
 
 #define FLB_INFO_FLAGS "@FLB_INFO_FLAGS@"
+
+#ifdef FLB_HAVE_TLS
+
+#ifdef FLB_SYSTEM_MACOS
```

Comment:
Could you abstract `FLB_MACOS_DEFAULT_CA`, `FLB_MACOS_CA_DIR`, `RHEL_DEFAULT_CA` and `"/etc/ssl/certs/"` in a platform agnostic way in this header (creating a new generic constant for the path that's currently hardcoded)?

That way we wouldn't need to have repeated conditional code in `openssl.c`

Reply:
Oh, it sounds good. Will do.

Reply:
Fixed in https://github.com/fluent/fluent-bit/pull/6035/commits/f3a42f9ac2a90ac0398daf0b2c2735c5528bd628.